src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -32,6 +33,7 @@ module.exports = Model.extend({
   },
 
   initialize: function () {
+    this.sync = BackboneCancelSync.bind(this);
```

Comment:
I don't know if the same behaviour happens in another data models or if it's something particular on this one.

Let me peek what other data models do.